server/app/views/applicant/NorthStarApplicantProgramSummaryView.java

```diff
@@ -51,6 +53,15 @@ public String render(Request request, Params params) {
             Optional.of(params.profile()),
             params.applicantPersonalInfo(),
             params.messages());
+
+    // Create a string such as "Program appplication summary - Pet Assistance Program"
+    String summarySubstring =
+        params.programType().equals(ProgramType.COMMON_INTAKE_FORM)
+            ? params.messages().at(MessageKey.TITLE_COMMON_INTAKE_SUMMARY.getKeyName())
+            : params.messages().at(MessageKey.TITLE_PROGRAM_SUMMARY.getKeyName());
+    String pageTitle = String.format("%s — %s", summarySubstring, params.programTitle());
```

Comment:
qq [non-blocking]- is the programTitle, the localized program name. We have had issue in the past that the admin name gets displayed.